gui/default/assets/lang/lang-ro-RO.json

```diff
@@ -442,6 +436,18 @@
     "full documentation": "toată documentaţia",
     "items": "obiecte",
     "seconds": "seconds",
+    "setup-auth": {
+        "notification": {
+            "body-1": "Username/Password has not been set for the GUI authentication. Please consider setting it up.",
+            "body-2": "If you want to prevent other users on this computer from accessing Syncthing and through it your files, consider setting up authentication.",
+            "title": "GUI Authentication: Set User and Password"
+        },
+        "warning": {
+            "body-1": "The Syncthing admin interface is configured to allow remote access without a password.",
+            "body-2": "This can easily give hackers access to read and change any files on your computer.",
+            "body-3": "Please set a GUI Authentication User and Password in the Settings dialog."
+        }
+    },
```

Comment:
These look wrong to me - these aren't translated, it's English in the "Romanian" translation. Looks like these shouldn't be in the file at all, as that likely shows up as false positives in tools indicating these are translated.

Looks like this file has a bunch of other strings like that too.

Reply:
See also:
- https://github.com/syncthing/syncthing/pull/9964